examples/Demo.Console.NET6/stylecop.json

```diff
@@ -0,0 +1,13 @@
+﻿{

```

Comment:
We can add stylecop as a link to this project so that we do not need to include it as a separate file here. We already have a global style cop file as the stylecop folder here which should be used here.

Reply:
Yeah, let's definitely avoid duplicating this config where we can. 😃